scripts/createImage.sh

```diff
@@ -17,9 +18,47 @@ dependencies=(
     target/dependency/armada-scala-client_${SCALA_BIN_VERSION}-0.1.0-SNAPSHOT.jar
 )
 
+spark_v3_dependencies=(
+    target/dependency/grpc-api-1.47.1.jar
+    target/dependency/grpc-core-1.47.1.jar
+    target/dependency/grpc-netty-1.47.1.jar
+    target/dependency/grpc-protobuf-1.47.1.jar
+    target/dependency/grpc-context-1.47.1.jar
+    target/dependency/grpc-stub-1.47.1.jar
+    target/dependency/guava-31.0.1-android.jar
+    target/dependency/failureaccess-1.0.1.jar
+    target/dependency/perfmark-api-0.25.0.jar
+)
+
+spark_v334_dependencies=(
```

Comment:
I think we can relax this to Spark 3.3
```suggestion
spark_v33_dependencies=(
```